subprojects/performance/contents/index.adoc

```diff
@@ -59,6 +59,24 @@ image::parallel-task-fast.png[title="Optimized parallel execution"]
 
 That’s the end of the quick wins. From here on out, improving your build performance will require some elbow grease. First, perhaps the most important step: finding out which bits of your build are slow and why.
 
+=== Enable file system watching
+
+Gradle talks a lot to the file system, especially when trying to resolve the state of the inputs and outputs of the build.
```

Comment:
"Gradle talks a lot to the file system" sounds a bit informal. Consider replacing it with something like "Gradle may perform a significant number of I/O operation on the file system in order to reduce build times, for example checking the state of task input and outputs for incremental builds".

Reply:
Would you mind making a suggestion?